### PR TITLE
Fix add to calendar links on VAOS

### PR DESCRIPTION
**Pull request by jbalboni**
## Description
This PR
- Fixes the formatting in the ICS file that was causing calendar apps to throw errors
- Fixes locations getting cutoff at the first comma
- Fixes ics validator error around line breaks needing to be CRLFs
- Fixes phone appointments to remove facility address
- Fixes incorrect duration calculation due to `.toDate()` call

How to test
- Make sure the va_online_scheduling_homepage_refresh flag is turned off
- Click on Add to calendar link on appointment list: http://localhost:3001/health-care/schedule-view-va-appointments/appointments/
- Open ICS file in a calendar app

## Testing done
Unit testing, local testing, used online ics validator tool

## Screenshots
Screen Shot 2021-02-02 at 4 57 11 PM

## Acceptance criteria
- [ ] Calendar links generate valid ICS files again

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/components/AddToCalendar.jsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ function formatDescription(description) {
   }
   chunked.push(restOfDescription);
 
-  return chunked.join('\r\n\t');
+  return chunked.join('\r\n\t').replace(/,/g, '\\,');
```
Comment (LanceGundersen) [inline src/applications/vaos/components/AddToCalendar.jsx:31]: I know it warned you but just want to be sure you saw this.

File: src/applications/vaos/components/AddToCalendar.jsx [hunk 2]
```diff
@@ -40,19 +40,21 @@ function generateICS(
...
-          END:VCALENDAR`;
+  return [
+    `BEGIN:VCALENDAR`,
+    `VERSION:2.0`,
+    `PRODID:VA`,
+    `BEGIN:VEVENT`,
+    `UID:${guid()}`,
+    `SUMMARY:${summary}`,
+    `${formatDescription(description)}`,
+    `LOCATION:${location?.replace(/,/g, '\\,')}`,
```
Comment (LanceGundersen) [inline src/applications/vaos/components/AddToCalendar.jsx:51]: Here as well

**Review by LanceGundersen — COMMENTED** [review]
This is the exchange calendar app.
image
image

**Review by narin — APPROVED** [review]
LGTM after our discussion if we're okay with that `time zone:Floating` message.

**Comment by jbalboni** [comment]
@narin I think I have that issue fixed now (along with other timezone issues).

**Pull request merged**